react_app/src/components/HomePage/AddCourseFormDialog.js

```diff
@@ -38,7 +44,48 @@ function AddCourseFormDialog(props) {
         <DialogTitle id="form-dialog-title">Add Course</DialogTitle>
         <DialogContent>
           <DialogContentText>
-            To add course, please enter details below:
+            For Admins: To add course, please enter details below:
+          </DialogContentText>
+          <TextField
+            autoFocus
+            required
+            margin="dense"
+            id="course_id"
+            label="CourseName"
+            onInput={ e=>courseName = e.target.value}
+            fullWidth
+          />
+          <TextField
+            required
+            margin="dense"
+            id="course_description"
+            label="Description"
+            onInput={ e=>courseDesc = e.target.value}
+            fullWidth
+          />
+        </DialogContent>
+        <DialogActions>
+            <Button onClick={handleClose} color="primary">
+              Cancel
+            </Button>
+            <Button onClick={() => handleAdminSubmit(courseName, courseDesc)} color="primary">
+              Submit
+            </Button>
```

Comment:
Similar to the comment above `{userRole === 'Admin' ? <Button onClick={() => handleAdminSubmit(courseName, courseDesc)} color="primary">
              Submit
            </Button> :
 <Button onClick={() => handleCustomerSubmit(courseName, courseDesc)} color="primary">
            Send to Admins
          </Button>}`

Reply:
done 👍